backend/app/processing/llm.py

```
@@ -210,7 +212,7 @@ def is_product_query(query: str, history_text: str = "") -> bool:
         
         headers = {
             "Authorization": f"Bearer {FREE_LLM_API_KEY}",
-            "Content-Type": "application/json"
+            "Content-Type": CONTENT_TYPE_JSON
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)